<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bitrateForm</class>
 <widget class="QWidget" name="bitrateForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="settingsButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>21</horstretch>
         <verstretch>21</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Stream settings</string>
       </property>
       <property name="toolTipDuration">
        <number>1</number>
       </property>
       <property name="whatsThis">
        <string>Stream settings</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qtWidgets/icons/icons.qrc">
         <normaloff>:/options.png</normaloff>:/options.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="exceptionLabel">
       <property name="text">
        <string> | no exceptions</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>17</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="playButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qtWidgets/icons/icons.qrc">
         <normaloff>:/play.png</normaloff>:/play.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pauseButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qtWidgets/icons/icons.qrc">
         <normaloff>:/pause.png</normaloff>:/pause.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>21</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../qtWidgets/icons/icons.qrc">
         <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="progressSlider">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::NoTicks</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qtWidgets/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
